src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -0,0 +1,259 @@
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.fedora2vault
+
+import nl.knaw.dans.common.lang.dataset.AccessCategory._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import nl.knaw.dans.lib.string._
+import nl.knaw.dans.pf.language.emd.types.EmdConstants.DateScheme
+import nl.knaw.dans.pf.language.emd.types._
+import nl.knaw.dans.pf.language.emd.{ EasyMetadataImpl, EmdRights }
+
+import scala.collection.JavaConverters._
+import scala.util.Try
+import scala.xml._
+
+object DDM extends DebugEnhancedLogging {
+  val schemaNameSpace: String = "http://easy.dans.knaw.nl/schemas/md/ddm/"
+  val schemaLocation: String = "https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd"
+  val dansLicense = "http://dans.knaw.nl/en/about/organisation-and-policy/legal-information/DANSLicence.pdf"
+  val cc0 = "http://creativecommons.org/publicdomain/zero/1.0"
+
+  def apply(emd: EasyMetadataImpl, audiences: Seq[String]): Try[Elem] = Try {
+    //    println(new EmdMarshaller(emd).getXmlString)
+
+    val dateMap: Map[String, Iterable[Elem]] = getDateMap(emd)
+    val dateCreated = dateMap("created")
+    val dateAvailable = {
+      val elems = dateMap("available")
+      if (elems.isEmpty) dateCreated
+      else elems
+    }
+   <ddm:DDM
+     xmlns:dc="http://purl.org/dc/elements/1.1/"
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xmlns:dct="http://purl.org/dc/terms/"
+     xmlns:dcx-dai="http://easy.dans.knaw.nl/schemas/dcx/dai/"
+     xmlns:dcx-gml="http://easy.dans.knaw.nl/schemas/dcx/gml/"
+     xmlns:gml="http://www.opengis.net/gml"
+     xmlns:abr="http://www.den.nl/standaard/166/Archeologisch-Basisregister/"
+     xmlns:ddm={schemaNameSpace}
+     xmlns:id-type="http://easy.dans.knaw.nl/schemas/vocab/identifier-type/"
+     xsi:schemaLocation={s"$schemaNameSpace $schemaLocation"}
+   >
+     <ddm:profile>
+       { emd.getEmdTitle.getDcTitle.asScala.map(bs => <dc:title xml:lang={ lang(bs) }>{ bs.getValue }</dc:title>) }
+       { emd.getEmdDescription.getDcDescription.asScala.map(bs => <dct:description xml:lang={ lang(bs) }>{ bs.getValue }</dct:description>) }
+       { /* instructions for reuse not specified as such in EMD */ }
+       { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator>{ bs.getValue }</dc:creator>) }
+       { emd.getEmdCreator.getEasCreator.asScala.map(author => <dcx-dai:creatorDetails>{ toXml(author)} </dcx-dai:creatorDetails>) }
+       { dateCreated.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
+       { dateAvailable.map(node =>  <ddm:available>{ node.text }</ddm:available>) }
+       { audiences.map(code => <ddm:audience>{ code }</ddm:audience>) }
+       <ddm:accessRights>{ emd.getEmdRights.getAccessCategory }</ddm:accessRights>
+     </ddm:profile>
+     <ddm:dcmiMetadata>
+       { emd.getEmdIdentifier.getDcIdentifier.asScala.filter(isDdmId).map(bi => <dct:identifier xsi:type={ idType(bi) }>{ bi.getValue }</dct:identifier>) }
```

Comment:
`easy-split-multi-deposit` uses `<dc:identifier xsi:type="...">...</dc:identifier>` here, as well as `<dc:identifier>...</dc:identifier>`. Is there any difference with using `dcterms` here?

Reply:
no, the dcterms element is a subproperty for the dc element. 
In general, it is recommended to use dcterms, as it is a bit more defined

Reply:
So it is OK as implemented?